lib/screen/common/alert_dialog.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/material.dart';
+import 'package:showcase_flutter_build_web_desktop/extension/context_extension.dart';
+
+import '../../gen/colors.gen.dart';
+import '../../theme/dimen.dart';
+
+Future<void> showAlertDialog({
+  required BuildContext context,
+  required VoidCallback onCloseClick,
+  required VoidCallback onPrintClick,
+  required VoidCallback onOkayClick,
+  required String receiptNumber,
+}) {
```

Comment:
This function will be re-used in many contexts. Consider updating it to make it more generic.

```suggestion
Future<void> showAlertDialog({
  required BuildContext context,
  required VoidCallback onCloseClick,
  required VoidCallback onPrintClick,
  required VoidCallback onOkayClick,
  required String title,
  required String body,
  required String firstButtonText,
  required String secondButtonText,
}) {
```

Reply:
Fixed in https://github.com/nimblehq/showcase-flutter-build-web-desktop/pull/24/commits/46ae752d4e4a033f4174b8df755745f41f167698 🙏